languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,169 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PAS_DE_VIRGULE" name="pas de virgule" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="[NJVA].*" postag_regexp="yes"><exception regexp="yes">^[a-z]$</exception></token>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">et|ou</token>
+                  </marker>
+                  <token postag="[DNJV].*" postag_regexp="yes"/>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\3</suggestion>
+              <example correction="et">Elle a acheté des radis<marker>, et</marker> des choux.</example>
+              <example>Les voyelles a, e, i, o, u<marker>, ou</marker> sont couramment utilisée dans la langue française.</example>
```

Comment:
utilisées?

Reply:
I corrected the pattern and added the punctuation that should be excluded.